tests/components/hunterdouglas_powerview/test_config_flow.py

```diff
@@ -23,7 +23,7 @@
 )
 
 DHCP_DISCOVERY_INFO = dhcp.DhcpServiceInfo(
-    hostname="Hunter Douglas Powerview Hub", ip="1.2.3.4"
+    hostname="Hunter Douglas Powerview Hub", ip="1.2.3.4", macaddress=None
```

Comment:
`None` doesn't match the type of the `macaddress` attribute. Probably make them optional if that's needed.

Reply:
@epenet Use a dummy mac address here.  It should never be `None`

Reply:
Something like `AA:BB:CC:DD:EE:FF` might be safer in case we somehow validate it